src/app/analytics/analytics.factory.js

```diff
@@ -251,12 +250,11 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, envService
       // Send GTM Advance Ecommerce event
       $window.dataLayer = $window.dataLayer || []
       $window.dataLayer.push({
-        event: 'remove-from-cart',
+        event: 'remove_from_cart',
         ecommerce: {
           currencyCode: 'USD',
-          remove: {
-            products: [generateProduct(item)]
-          }
+          value: item.amount.toFixed(2),
```

Comment:
I would be curious how this is working, and not needing to be `AMOUNT`. I know the old code is doing it, but maybe it is broken?

Reply:
I guess it wouldn't be broken, otherwise the cart would not show the data since the `cart.tpl.html` is using lowercase.

Reply:
I see where the transformation is, in `cart.service.js` `handleCartResponse()`